pkg/view/project.go

```diff
@@ -38,6 +38,73 @@ type ProjectHead struct {
 	Avatar      string `json:"avatar"`
 }
 
+type UpdateProjectStatusResponse struct {
+	Data ProjectData `json:"data"`
+}
+
+func ToUpdateProjectStatusResponse(p *model.Project) ProjectData {
```

Comment:
Can you return to another data structure? `ProjectData` here, should be used for listing. So I will update the name of this struct later. Btw, you can create a new one and return the same thing with model.Project